src/scripts/resource-channels.js

```diff
@@ -78,4 +79,10 @@ function resourcesMuteMessage(message, client) {
 		.catch(console.error);
 }
 
+function checkIfResource(message) {
+	if (message.attachments.size) return true; // Ignores attachments
+	if (message.content.includes("http")) return true; // Ignores all links
+	return false;
+}
+
```

Comment:
I also added a check for images. I noticed quite a few users were uploading images in addition to links.